### PR TITLE
DIT and DAT courses

### PR DESCRIPTION
**Pull request by 20Attila03**

### CODE REVIEW

File: exams/DAT050/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# DAT050 - Objektorientered programmering
+Welcome to the DAT050 - Objektorientered programmering, where we've compiled past exams and student answers to assist in your preparation for this course.
+
+Here’s what we have so far:
+
+|    Date    | Questions | Answers |
```
Comment (michalspano) [inline exams/DAT050/README.md:6]: This is cool, we should add the table to the already existing courses too.

**Comment by michalspano** [comment]
Why was this PR closed? Will there be a new PR scheduled? 🤔

**Comment by michalspano** [comment]
Anyhow, I saw the last changes... pretty cool! 👍🏿

**Comment by michalspano** [comment]
@ me when the updated PR is up.

**Comment by 20Attila03** [comment]
It will be up soon yeah :)

**Pull request closed without merging**